logger.js

```diff
@@ -104,10 +104,10 @@ module.exports = (externalLogger, logFolder, logFilePrefix = "installer")=> {
     setDisplaySettings(settings) {
       if (externalLogger) {externalLogger.setDisplaySettings(settings);}
     },
-    external(evt, detail) {
-      appendToLog(evt, detail);
+    external(evt, detail, table) {
+      appendToLog(table || evt, detail);
 
-      if (externalLogger) {externalLogger.log(evt, detail);}
+      if (externalLogger) {externalLogger.log(table || evt, detail);}
```

Comment:
@jungeunyoon This shouldn't conditionally apply *table* or *evt*.  External logger `log()` should accept *evt*, *detail*, and *table*.